test/user.js

```diff
@@ -127,4 +127,116 @@ describe('users', () => {
         });
     });
   });
+
+  describe('/users/sessions/ POST', () => {
+    it('should login an user without problems', done => {
+      chai
+        .request(server)
+        .post('/users/')
+        .send({
+          firstName: 'pepito',
+          lastName: 'perez',
+          email: 'pepito.perez@wolox.com',
```

Comment:
Make a function for create users, check this in all test file

Reply:
like as?